src/components/Comparisons/ComparisonsModal.jsx

```diff
@@ -41,11 +41,30 @@ const TL = styled.td`
 const TR = styled.td`
   text-align: right;
 `
+const Span = styled.span`
+`
 
-const ComparisonsModal = (props) => {
+function ComparisonsModal (props) {
   if (!props.show) {
     return null;
   }
+  const [characteristics, setCharacteristics] = useState([]);
+
+  //props.prod = current product being viewed
+  //props.item = related item clicked on
+
+  //sets characteristics to the features of both the current product being viewed and the related item the user clicked on (with no duplicates)
+  useEffect(() => {
+    var combinedChar = props.prod.features.concat(props.item.features);
```

Comment:
Somewhere where you concat these together, it seems to be leaving out spaces which you can see in the modal. Need to find where that's happening and adjust (for now maybe just add it as a refactoring ticket?)

Reply:
Waittt how are you able to see this without running the code? haha 

So I'm not sure why but the API sends back some of the values with no spaces between the words.. But yes will have to edit that in a new ticket...

![Screen Shot 2022-09-01 at 4 32 26 PM](https://user-images.githubusercontent.com/94999450/188016864-bdbc17f9-82d1-4552-aa05-88d4a0ca76aa.jpg)